src/components/sponsorsComponents/CurrentSponsors.astro

```diff
@@ -0,0 +1,103 @@
+---
+import { currentSponsors } from '../../data/currentSponsors.ts'
+import SponsorCategory from './SponsorCategory.astro'
+---
+<div class="outer-container">
+    <h1 class="title">Sponsors</h1>
+    <p class="intro">We rely on your support to succeed.</p>
+    <p class="intro">Thank you to our generous sponsors who make our science happen.</p>
+
+    <div class="sponsors">
+        {
+            currentSponsors.map((sponsor)=> (
+                <>
+                    <p class="sponsor-title">PLATINUM</p>
+                    <SponsorCategory 
+                        name="" 
+                        icons={sponsor.icons}
+                        path="../../../static/sponsor-icons/"/>
+                </>
+            ))
+        }
+    </div>
+    <a href="#want-to-sponsor-us"><button class="sponsor-us"><p>Support Us</p></button></a>
+</div>
+
+<style>
+    * {
+        --sponsor-image-height: 6rem;
+        --sponsor-container-min-height:40rem;
+        margin:0;
+    }
+
+    .title {
+        font-size:4.688rem;
+        margin-bottom:1rem;
+        display:block;
+    }
+
+    .intro {
+        font-size:1.4rem;
+        font-weight:500;
+    }
+
+    .outer-container{
+        width:100%;
+        padding:6rem;
+        background: var(--color-primary-green);
+        display:flex;
+        align-items: start;
+        flex-direction:column;
+        border-bottom: 2px solid black;
+    }
+
+    .sponsor-title {
+        text-align:center;
+        font-size:1.45rem;
+        font-weight:lighter;
+        margin: 3rem auto 1.5rem auto;
+    }
+
+    .sponsors {
+        min-height:var(--sponsor-container-min-height);
```

Comment:
could you remove the empty space for now? we'll probably have an updated list of current sponsors soon but for now the empty space looks kind of weird.

![Screenshot 2025-06-22 at 16 48 32](https://github.com/user-attachments/assets/cad79c3d-4536-40aa-882f-f1db3eefa45c)